lib/galaxy/schema/types.py

```diff
@@ -1,12 +1,16 @@
 from datetime import datetime
+from typing import Union
 
+from pydantic import AnyUrl
 from pydantic.datetime_parse import parse_datetime
 from typing_extensions import Literal
 
 # Relative URLs cannot be validated with AnyUrl, they need a scheme.
 # Making them an alias of `str` for now
 RelativeUrl = str
 
+AbsoluteOrRelativeUrl = Union[AnyUrl, RelativeUrl]
```

Comment:
```suggestion
AbsoluteOrRelativeUrl = RelativeUrl
```

Reply:
I was hoping this was validating the URL when it was absolute and then fallback to relative (str) when it failed... but I guess it will always fallback to str so maybe we can just remove `AbsoluteOrRelativeUrl` and use directly `RelativeUrl`?

Reply:
Right, that is what happens, but we don't know that and it'll never catch anything ... htt:/url.com is going to be a valid string.